### ISSUE TITLE
quiet mode to make bot responses less verbose for large chatrooms

### ISSUE THREAD

**Issue by MoralCode** [fedora-infra/maubot-fedora#89]
Matrix was used as the event platform for the F40 release party. As people discovered the cookie-giving features, chat got a little spammy, especially with the `!cookie count` command.

![Screenshot from 2024-05-24 09-49-36](https://github.com/fedora-infra/maubot-fedora/assets/17362949/1a9141a2-de0c-4808-9d6e-e82caeac2084)

I think it may be nice to have a "quiet" mode that does something like:

- move bot replies to DMs or another room
- reduce the verbosity of longer replies
- rate limits replies or batches them into one message
.etc

**Comment by ryanlerch** [comment]
hmmm, not sure if this is worth the hassle TBH.

1. People can DM zodbot itself and use these commands if they want.
2. the only command we really have that outputs all this is that one particular command. (and only for a handful of users too @nirik has been around for a while)

**Comment by MoralCode** [comment]
> hmmm, not sure if this is worth the hassle TBH.

yeah i think you might be right. Been trying to figure out how to avoid doing too much Yak shaving lately haha. Apologies for the extraneous issue!
